app/views/providers/proceeding_loop/substantive_defaults/show.html.erb

```diff
@@ -14,23 +14,25 @@
     <p><strong><%= t(".scope_header") %></strong> <%= form.object.substantive_scope_limitation_meaning %></p>
     <p><strong><%= t(".scope_description_header") %></strong> <%= form.object.substantive_scope_limitation_description %></p>
 
-    <%= form.govuk_radio_buttons_fieldset :accepted_substantive_defaults,
-                                          inline: false,
-                                          legend: { size: "m", tag: "h2", text: t(".question") } do %>
+    <% if !@proceeding.special_childrens_act? %>
+      <%= form.govuk_radio_buttons_fieldset :accepted_substantive_defaults,
+                                            inline: false,
+                                            legend: { size: "m", tag: "h2", text: t(".question") } do %>
 
-      <%= form.hidden_field :substantive_level_of_service %>
-      <%= form.hidden_field :substantive_level_of_service_name %>
-      <%= form.hidden_field :substantive_level_of_service_stage %>
-      <%= form.hidden_field :substantive_scope_limitation_meaning %>
-      <%= form.hidden_field :substantive_scope_limitation_description %>
-      <%= form.hidden_field :substantive_scope_limitation_code %>
+        <%= form.hidden_field :substantive_level_of_service %>
+        <%= form.hidden_field :substantive_level_of_service_name %>
+        <%= form.hidden_field :substantive_level_of_service_stage %>
+        <%= form.hidden_field :substantive_scope_limitation_meaning %>
+        <%= form.hidden_field :substantive_scope_limitation_description %>
+        <%= form.hidden_field :substantive_scope_limitation_code %>
```

Comment:
Are these hidden values considered part of the default, highest level of service and scope? If yes, should they be present regardless if SCA or not?